build/patches/expose-offer-extmap-allow-mixed.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/sdk/android/api/org/webrtc/PeerConnection.java b/sdk/android/api/org/webrtc/PeerConnection.java
+index bf0a2e9441..aa44dbae05 100644
+--- a/sdk/android/api/org/webrtc/PeerConnection.java
++++ b/sdk/android/api/org/webrtc/PeerConnection.java
+@@ -555,6 +555,14 @@ public class PeerConnection {
+      */
+     @Nullable public CryptoOptions cryptoOptions;
+ 
++    /**
++     * Configure if we should include the SDP attribute extmap-allow-mixed in
++     * our offer. Although we currently do support this, it's not included in
++     * our offer by default due to a previous bug that caused the SDP parser to
++     * abort parsing if this attribute was present. This is fixed in Chrome 71.
++     */
++     public boolean offerExtmapAllowMixed;
++
+     /**
+      * An optional string that if set will be attached to the
+      * TURN_ALLOCATE_REQUEST which can be used to correlate client
+@@ -605,6 +613,9 @@ public class PeerConnection {
+       useMediaTransport = false;
+       useMediaTransportForDataChannels = false;
+       cryptoOptions = null;
++      // TODO(webrtc:9985): Change default to true once sufficient time has
++      // passed.
```

Comment:
I assume this webrtc bug(?) reference is copy-pasted from the header file, right? https://bugs.chromium.org/p/webrtc/issues/detail?id=9985 seems to be inaccessible.

Reply:
The comment is from the native variant, so I just copied it.